sympy/solvers/inequalities.py

```diff
@@ -396,7 +396,17 @@ def solve_univariate_inequality(expr, gen, relational=True):
 
     """
 
-    from sympy.solvers.solvers import solve, denoms
+    from sympy.solvers.solvers import denoms
+    from sympy.solvers.solveset import solveset
+
+    def solve_replacement(f_x, x):
```

Comment:
You could just capture `solve` as `sympy_solve=solve` and then `def solve` instead of `def solve_replacement` using `sympy_solve` internally so no other changes would be necessary in the file.